api/api/openapi.bundle.yaml

```diff
@@ -2350,8 +2350,10 @@ components:
         experiment_mappings:
           items:
             $ref: '#/components/schemas/EnsemblerStandardConfig_experiment_mappings'
+          nullable: true
           type: array
         route_name_path:
+          nullable: true
```

Comment:
These two `nullable` conditions have to be added to force the autogenerated openapi classes to accept `None` values for both `experiment_mappings` and `route_name_path`.